torch/_C/__init__.pyi.in

```diff
@@ -22,6 +22,8 @@ import builtins
 from . import _nn as _nn
 from . import _onnx as _onnx
 from . import _VariableFunctions as _VariableFunctions
+from . import _lazy as _lazy
```

Comment:
is this some weird python thing?
why can't we just omit `as _lazy`?

Reply:
hmm, I have no idea, i just copied what others did.